src/app/modules/item/item-routing.module.ts

```diff
@@ -43,6 +44,12 @@ import { ItemLogViewComponent } from './pages/item-log-view/item-log-view.compon
         {
           path: 'edit',
           component: ItemEditComponent,
+          children: [
+            {
+              path: '',
+              component: ItemEditContentComponent
+            }
+          ]
```

Comment:
Your code does not use routing to choose tab so there is no routing to be configured to your component.

Reply:
Fixed